industrial_ci/src/tests/debians.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Copyright (c) 2021, Mathias Lüdtke
+# All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Based on https://github.com/ros-industrial/industrial_ci/issues/697#issuecomment-876293987
+
+function prepare_debians() {
+  true
+}
+
+function build_and_install_debian() (
+    set -e
+    local pkg_path=$1
+    shift
+    cd "$pkg_path"
+    ici_quiet_echo bloom-generate rosdebian --ros-distro="$ROS_DISTRO" --debian-inc="ici~"
+    ici_quiet_echo dpkg-buildpackage -b -uc -us
+    mv ../*.{buildinfo,changes,deb} .
```

Comment:
superflous

Reply:
Why?

Reply:
please give reasons why you need this (instead of just using `..` in the path for dpkg/apt below.

Reply:
To only install a single *.deb file, and not all of them every time.

Reply:
hmkay, maybe add a comment then.